src/web/routes.rs

```diff
@@ -240,6 +276,10 @@ pub(super) fn build_routes() -> Routes {
         super::rustdoc::rustdoc_html_server_handler,
     );
 
+    for prefix in INTERNAL_PREFIXES {
+        routes.add_internal_page_prefix(prefix);
```

Comment:
when removing the internal routes from the iron handler, the prefixes were not registered any more.

Here I just add fixed internal prefixes for now, we can rethink that when all handlers are migrated.